src/index.html

```diff
@@ -26,10 +26,14 @@
     <!-- ! DO NOT REMOVE THIS COMMENT, WE NEED ITS CLOSING MARKER -->
   </head>
 
-  <body>
+  <body style="background: url('./src/assets/img/buddhanexus.jpg') cover;">
+    <div class="head">
+        <img src="./src/assets/img/buddha-nexus.png" alt="Buddha Nexus">
+    </div>
+
     <vaadin-app-layout>
       <vaadin-drawer-toggle slot="navbar"></vaadin-drawer-toggle>
-      <h1 slot="navbar" class="header-title">Buddhanexus</h1>
+      <h1 slot="navbar" class="header-title"><img src="./src/assets/img/buddhanexus.jpg" class="buddhanexus" alt="Buddha Nexus"></h1>
```

Comment:
naming: `buddhanexus` is not very descriptive as a class name for this `image` element. How about `buddhanexus-logo`?